101-custom-rp-with-function/azuredeploy.json

```diff
@@ -128,35 +125,33 @@
                 "[resourceId('Microsoft.Web/sites', parameters('funcName'))]"
             ],
             "properties": {
-                "actions": [
-                    {
-                        "name": "ping",
-                        "routingType": "Proxy",
-                        "endpoint": "[concat('https://', reference(resourceId('Microsoft.Web/sites', parameters('funcName')), '2018-02-01').defaultHostName, '/api/{requestPath}')]"
-                    }
-                ],
                 "resourceTypes": [
                     {
-                        "name": "users",
-                        "routingType": "Proxy,Cache",
-                        "endpoint": "[concat('https://', reference(resourceId('Microsoft.Web/sites', parameters('funcName')), '2018-02-01').defaultHostName, '/api/{requestPath}')]"
+                        "name": "customResources",
+                        "routingType": "Proxy",
+                        "endpoint": "[listSecrets(resourceId('Microsoft.Web/sites/functions', parameters('funcName'), 'HttpTrigger1'), '2018-02-01').trigger_url]"
                     }
                 ]
             }
         },
         {
-            "type": "Microsoft.CustomProviders/resourceProviders/users",
-            "name": "[concat(parameters('funcName'), '/addUser')]",
+            "type": "Microsoft.CustomProviders/resourceProviders/customResources",
+            "name": "[concat(parameters('funcName'), '/', parameters('azureCustomResourceName'))]",
             "apiVersion": "2018-09-01-preview",
             "location": "[parameters('location')]",
             "dependsOn": [
                 "[resourceId('Microsoft.CustomProviders/resourceProviders', parameters('funcName'))]"
             ],
             "properties": {
-                "FullName": "[parameters('userName')]",
-                "Location": "[parameters('userLocation')]"
+                "hello": "world",
+                "myCustomProperty": "[parameters('myCustomPropertyValue')]"
             }
         }
-
-    ]
+    ],
+    "outputs": {
+        "CustomResource": {
+            "type": "string",
+            "value": "[reference(resourceId('Microsoft.CustomProviders/resourceProviders/customResources', parameters('funcName'), parameters('azureCustomResourceName')), '2018-09-01-preview', 'Full').resourceId]"
```

Comment:
isn't this just a resourceId (so we could just use the resourceId function?)

Reply:
I removed the outputs. I think that it doesn't provide that much value.